SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -2792,6 +2812,19 @@ void CFEASolver::PredictStruct_Displacement(CGeometry *geometry, CConfig *config
 
 }
 
+void CFEASolver::PredictStruct_Velocity(CGeometry *geometry, CConfig *config) {
+
+  /*--- To nPointDomain: we need to communicate the predicted solution after setting it. ---*/
+  SU2_OMP_PARALLEL_(for schedule(static,omp_chunk_size))
+  for (unsigned long iPoint=0; iPoint < nPointDomain; iPoint++) {
+    nodes->SetSolution_Vel_Pred(iPoint);
+  }
+
+  InitiateComms(geometry, config, SOLUTION_VEL_PRED);
+  CompleteComms(geometry, config, SOLUTION_VEL_PRED);
+
+}
```

Comment:
What is the implication of using a higher order predictor for the displacement and a 0 order (possibly inconsistent) for the velocity?

If there is no implication we can avoid all the Set/GetSolution_Vel_Pred and use the velocity directly.

Reply:
Can you try running the loop to nPoint and not doing the communications? (in principle it should work)

Reply:
Thanks for the review Pedro. I'm not sure about the mix of predictor orders. I've only used and tested with zero order displacement and velocity. I know there is more code to create the set/get calls, but I followed the same procedure as the displacement so that it can be adapted if needed in the future.

The change to nPoint worked and I could remove the additional communication statements here.

Reply:
Alright let's leave it for now, good to have options.